go.mod

```diff
@@ -169,7 +169,8 @@ require (
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-replace github.com/cosmos/cosmos-sdk => github.com/provenance-io/cosmos-sdk v0.46.10-pio-3
+// TODO: Update this to a release version once one is cut.
+replace github.com/cosmos/cosmos-sdk => github.com/provenance-io/cosmos-sdk v0.46.3-pio-4.0.20230425212412-d917960f5c80
```

Comment:
This needs to be changed to a release version (`v0.46.10-pio-4`?) before this PR can be merged.
That can't happen until https://github.com/provenance-io/cosmos-sdk/pull/568 has merged.

Reply:
Done.